bundles/GoogleMarketingBundle/src/CustomReport/Adapter/Analytics.php

```diff
@@ -14,9 +14,10 @@
  *  @license    http://www.pimcore.org/license     GPLv3 and PCL
  */
 
-namespace Pimcore\Model\Tool\CustomReport\Adapter;
+namespace Pimcore\Bundle\GoogleMarketingBundle\CustomReport\Adapter;
 
 use Pimcore\Bundle\CustomReportsBundle\Tool\Adapter\AbstractAdapter;
+use Pimcore\Bundle\GoogleMarketingBundle\Api\Api;
```

Comment:
If i uninstall the bundle and had Reports with `custom_report_adapter_analytics` as source definition, and try to open them, it would crash
![image](https://user-images.githubusercontent.com/6014195/220586915-da8094c8-3d84-48af-9cc5-503a520f04ae.png)

Maybe more as a follow-up for Custom Reports

Reply:
One should do the data cleanup before uninstalling the bundle.